src/vs/workbench/contrib/chat/browser/chatContentParts/chatToolInvocationPart.ts

```diff
@@ -134,10 +135,12 @@ class ChatToolInvocationSubPart extends Disposable {
 		@ILanguageService private readonly languageService: ILanguageService,
 		@IContextKeyService private readonly contextKeyService: IContextKeyService,
 		@ILanguageModelToolsService private readonly languageModelToolsService: ILanguageModelToolsService,
+		@IAccessibilityService private readonly _accessibilityService: IAccessibilityService
 	) {
 		super();
 
 		if (toolInvocation.kind === 'toolInvocation' && toolInvocation.confirmationMessages) {
+			this._accessibilityService.alert(toolInvocation.confirmationMessages.title);
```

Comment:
This is where the confirmation renders, so eg it would trigger multiple times as you scroll up and down, or it would not trigger at all if you are not scrolled down when the confirmation comes in. It's fine with me if you like that logic, but just checking

Reply:
oh, maybe better to go with the other place then.